platforms-serverless/serverless-framework-lambda/test.ts

```diff
@@ -10,8 +10,8 @@ async function main() {
 
   const actual = (data.$response.data as any).Payload
   const binaryString = process.env.PRISMA_CLIENT_ENGINE_TYPE === 'binary'
-    ? `,"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","schema.prisma"]`
-    : `,"files":["deno","edge.d.ts","edge.js","index-browser.js","index.d.ts","index.js","libquery_engine-rhel-openssl-1.0.x.so.node","package.json","schema.prisma"]`
+    ? `,"files":["index-browser.js","index.d.ts","index.js","package.json","query-engine-rhel-openssl-1.0.x","runtime","schema.prisma"]`
+    : `,"files":["index-browser.js","index.d.ts","index.js","libquery_engine-rhel-openssl-1.0.x.so.node","package.json","runtime","schema.prisma"]`
```

Comment:
Why the difference here?

Reply:
Custom outputs include runtime files, not the default output